src/Control/Monad/Aff.purs

```diff
@@ -75,17 +76,22 @@ cancelWith aff c = runFn3 _cancelWith nonCanceler aff c
 -- | If you do need to handle exceptions, you can use `runAff` instead, or
 -- | you can handle the exception within the Aff computation, using
 -- | `catchError` (or any of the other mechanisms).
-launchAff :: forall e a. Aff e a -> Eff (err :: EXCEPTION | e) Unit
-launchAff = runAff throwException (const (pure unit)) <<< liftEx
-  where
-  liftEx :: Aff e a -> Aff (err :: EXCEPTION | e) a
-  liftEx = _unsafeInterleaveAff
+launchAff :: forall e a. Aff (err :: EXCEPTION | e) a -> Eff (err :: EXCEPTION | e) Unit
```

Comment:
Why not modify this to return a canceller, to? User can always ignore a canceler if it's not necessary. Also this would allow you to delete the helper function `runAff_`.